common/config/rush/pnpm-lock.yaml

```diff
@@ -9291,17 +9254,16 @@ packages:
   file:projects/container-registry.tgz:
     dependencies:
       '@azure/core-tracing': 1.0.0-preview.13
-      '@azure/ms-rest-nodeauth': 3.0.10
```

Comment:
This PR is needed to remove this dep from container registry deps list because it is currently depending on a vulnerable package.